### DISCUSSION TITLE
Scroll up and down with thumb buttons continuously

### DISCUSSION THREAD

**Discussion by skareel** [linearmouse/linearmouse#725] [General]
I have linear mouse setup where the thumb back and forward buttons are set to scroll up and scroll down instead. However, they only do that by clicking over and over again instead of being able to hold down the thumb buttons and having it scroll continously until I let off. 

Can this feature be implemented into linearmouse?

**Comment by mail495sofar** [comment]
Yes. I want to scroll by axis with mac trackpad while holding a key

**Reply by mail495sofar** [reply]
I mentioned to isolate horizontal scrolling using Shift+scroll with trackpad (which without Shift just does usual all around scroll).
